cmd/lotus-miner/init.go

```diff
@@ -218,13 +216,6 @@ var initCmd = &cli.Command{
 				return err
 			}
 
-			// load bundles
-			bs := blockstore.NewMemory()
-
-			if err := bundle.FetchAndLoadBundles(ctx, bs, build.BuiltinActorReleases); err != nil {
-				return err
-			}
```

Comment:
We used to use this for the side-effect of initializing the registry in `chain/actors`, but that's now always initialized on init.